[EI61] Exception when starting the broker as a windows service

**Steps**
Copy wrapper.conf file inside the EI/wso2/broker/bin/yasjw to the YAJSW/conf directory
Start as "runConsole.bat"

**Observation**
Failed to start the service due to following exception.

```
C:\Users\wso2\Downloads\alpha\yajsw-stable-11.03\bat>runConsole.bat

C:\Users\wso2\Downloads\alpha\yajsw-stable-11.03\bat>cd C:\Users\wso2\Downloads\alpha\yajsw-stable-11.03\bat\

C:\Users\wso2\Downloads\alpha\yajsw-stable-11.03\bat>call setenv.bat
"java" -Xmx30m -Djna_tmpdir="C:\Users\wso2\Downloads\alpha\yajsw-stable-11.03\bat\/../tmp" -jar "C:\Users\wso2\Downloads\alpha\yajsw-stable-11.03\bat\/../wrapper.jar" -c "C:\Users\wso2\Downloads\alpha\yajsw-stable-11.03\bat\/../conf/wrapper.conf"
YAJSW: yajsw-stable-11.03
OS   : Windows 10/10.0/x86
JVM  : Oracle Corporation/1.8.0_121
Mar 31, 2017 10:49:52 AM org.apache.commons.vfs2.VfsLog info
INFO: Using "C:\Users\wso2\AppData\Local\Temp\vfs_cache" as temporary files store.
platform null
platform null
WARNING|wrapper|"WSO2 Carbon"|17-03-31 10:49:52|YAJSW: yajsw-stable-11.03
WARNING|wrapper|"WSO2 Carbon"|17-03-31 10:49:52|OS   : Windows 10/10.0/x86
WARNING|wrapper|"WSO2 Carbon"|17-03-31 10:49:52|JVM  : Oracle Corporation/1.8.0_121
Using the autodetected NIO constraint level: 0
INFO|wrapper|"WSO2 Carbon"|17-03-31 10:49:52|ahessian jmx service bound to port 15002
INFO|wrapper|"WSO2 Carbon"|17-03-31 10:49:53|spawned system tray icon process with pid 2104
INFO|wrapper|"WSO2 Carbon"|17-03-31 10:49:53|set state IDLE->STARTING
INFO|wrapper|"WSO2 Carbon"|17-03-31 10:49:53|Controller State: UNKNOWN -> WAITING
Mar 31, 2017 10:49:53 AM org.rzo.yajsw.wrapper.FileUtils getFiles
WARNING: No files found for C:\Program Files (x86)\Java\jre1.8.0_121\lib\tools.jar
INFO|wrapper|"WSO2 Carbon"|17-03-31 10:49:53|working dir C:\Users\wso2\Desktop\EI610\ALPHA\wso2ei-6.1.0-ALPHA\wso2\broker
INFO|wrapper|"WSO2 Carbon"|17-03-31 10:49:53|create script: scripts/trayMessage.gv
INFO|wrapper|"WSO2 Carbon"|17-03-31 10:49:53|found script scripts/trayMessage.gv
INFO|wrapper|"WSO2 Carbon"|17-03-31 10:49:53|started process with pid 764
INFO|wrapper|"WSO2 Carbon"|17-03-31 10:49:53|set state STARTING->RUNNING
WARNING|wrapper|"WSO2 Carbon"|17-03-31 10:49:54|[id: 0x01a7e526, /127.0.0.1:56386 => /127.0.0.1:15002] connected
INFO|764/0|"WSO2 Carbon"|17-03-31 10:49:55|[INFO] StandardFileSystemManager - Using "C:\Users\wso2\Desktop\EI610\ALPHA\wso2ei-6.1.0-ALPHA\wso2\broker\..\tmp\vfs_cache" as temporary files store.
INFO|764/0|"WSO2 Carbon"|17-03-31 10:49:55|platform null
INFO|764/0|"WSO2 Carbon"|17-03-31 10:49:56|Controller State: WAITING -> ESTABLISHED
INFO|764/0|"WSO2 Carbon"|17-03-31 10:49:56|Controller State: ESTABLISHED -> LOGGED_ON
INFO|764/0|"WSO2 Carbon"|17-03-31 10:49:57|[2017-03-31 10:49:57,441]  WARN {org.wso2.carbon.server.extensions.DropinsBundleDeployer} -  Ignoring the deployment of bundle: ..\dropins\bcprov_jdk15on_1.56_1.0.0.jar, because it is already available in the system: ..\..\..\dropins\bcprov_jdk15on_1.56_1.0.0.jar. Bundle-SymbolicName and Bundle-Version headers are identical
INFO|764/0|"WSO2 Carbon"|17-03-31 10:49:57|[2017-03-31 10:49:57,441]  WARN {org.wso2.carbon.server.extensions.DropinsBundleDeployer} -  Ignoring the deployment of bundle: ..\dropins\BPMNAnalyticsUDF_1.0.0.jar, because it is already available in the system: ..\..\..\dropins\BPMNAnalyticsUDF_1.0.0.jar. Bundle-SymbolicName and Bundle-Version headers are identical
INFO|764/0|"WSO2 Carbon"|17-03-31 10:49:57|[2017-03-31 10:49:57,441]  WARN {org.wso2.carbon.server.extensions.DropinsBundleDeployer} -  Ignoring the deployment of bundle: ..\dropins\bpmn_sample_jars_6.1.0_ALPHA_1.0.0.jar, because it is already available in the system: ..\..\..\dropins\bpmn_sample_jars_6.1.0_ALPHA_1.0.0.jar. Bundle-SymbolicName and Bundle-Version headers are identical
INFO|764/0|"WSO2 Carbon"|17-03-31 10:49:57|[2017-03-31 10:49:57,441]  WARN {org.wso2.carbon.server.extensions.DropinsBundleDeployer} -  Ignoring the deployment of bundle: ..\dropins\esapi_2.0.1_1.0.0.jar, because it is already available in the system: ..\..\..\dropins\esapi_2.0.1_1.0.0.jar. Bundle-SymbolicName and Bundle-Version headers are identical
INFO|764/0|"WSO2 Carbon"|17-03-31 10:49:57|[2017-03-31 10:49:57,441]  WARN {org.wso2.carbon.server.extensions.DropinsBundleDeployer} -  Ignoring the deployment of bundle: ..\dropins\org.wso2.carbon.logging.propfile_1.0.0.jar, because it is already available in the system: ..\..\..\dropins\org.wso2.carbon.logging.propfile_1.0.0.jar. Bundle-SymbolicName and Bundle-Version headers are identical
INFO|764/0|"WSO2 Carbon"|17-03-31 10:49:57|[2017-03-31 10:49:57,441]  WARN {org.wso2.carbon.server.extensions.DropinsBundleDeployer} -  Ignoring the deployment of bundle: ..\dropins\process_tools_6.1.0_ALPHA_1.0.0.jar, because it is already available in the system: ..\..\..\dropins\process_tools_6.1.0_ALPHA_1.0.0.jar. Bundle-SymbolicName and Bundle-Version headers are identical
INFO|764/0|"WSO2 Carbon"|17-03-31 10:49:57|[2017-03-31 10:49:57,441]  WARN {org.wso2.carbon.server.extensions.DropinsBundleDeployer} -  Ignoring the deployment of bundle: ..\dropins\sample_utils_6.1.0_ALPHA_1.0.0.jar, because it is already available in the system: ..\..\..\dropins\sample_utils_6.1.0_ALPHA_1.0.0.jar. Bundle-SymbolicName and Bundle-Version headers are identical
INFO|764/0|"WSO2 Carbon"|17-03-31 10:49:57|[2017-03-31 10:49:57,441]  WARN {org.wso2.carbon.server.extensions.DropinsBundleDeployer} -  Ignoring the deployment of bundle: ..\dropins\sqljdbc42_1.0.0.jar, because it is already available in the system: ..\..\..\dropins\sqljdbc42_1.0.0.jar. Bundle-SymbolicName and Bundle-Version headers are identical
INFO|764/0|"WSO2 Carbon"|17-03-31 10:50:06|[2017-03-31 10:50:06,316] []  INFO - CarbonCoreActivator Starting WSO2 Carbon...
INFO|764/0|"WSO2 Carbon"|17-03-31 10:50:06|[2017-03-31 10:50:06,331] []  INFO - CarbonCoreActivator Operating System : Windows 10 10.0, x86
INFO|764/0|"WSO2 Carbon"|17-03-31 10:50:06|[2017-03-31 10:50:06,331] []  INFO - CarbonCoreActivator Java Home        : C:\Program Files (x86)\Java\jre1.8.0_121
INFO|764/0|"WSO2 Carbon"|17-03-31 10:50:06|[2017-03-31 10:50:06,331] []  INFO - CarbonCoreActivator Java Version     : 1.8.0_121
INFO|764/0|"WSO2 Carbon"|17-03-31 10:50:06|[2017-03-31 10:50:06,331] []  INFO - CarbonCoreActivator Java VM          : Java HotSpot(TM) Client VM 25.121-b13,Oracle Corporation
INFO|764/0|"WSO2 Carbon"|17-03-31 10:50:06|[2017-03-31 10:50:06,331] []  INFO - CarbonCoreActivator Carbon Home      : C:\Users\wso2\Desktop\EI610\ALPHA\wso2ei-6.1.0-ALPHA\wso2\broker
INFO|764/0|"WSO2 Carbon"|17-03-31 10:50:06|[2017-03-31 10:50:06,331] []  INFO - CarbonCoreActivator Java Temp Dir    : C:\Users\wso2\Desktop\EI610\ALPHA\wso2ei-6.1.0-ALPHA\wso2\broker\..\tmp
INFO|764/0|"WSO2 Carbon"|17-03-31 10:50:06|[2017-03-31 10:50:06,331] []  INFO - CarbonCoreActivator User             : wso2, en-US, Asia/Colombo
INFO|764/0|"WSO2 Carbon"|17-03-31 10:50:06|[2017-03-31 10:50:06,612] []  WARN - ValidationResultPrinter The running OS : Windows 10 is not a tested Operating System for running WSO2 Carbon
INFO|764/0|"WSO2 Carbon"|17-03-31 10:50:06|[2017-03-31 10:50:06,628] []  INFO - GoogleTokenGenDSComponent Activating GoogleTokengen DS component
INFO|764/0|"WSO2 Carbon"|17-03-31 10:50:06|[2017-03-31 10:50:06,800] []  INFO - KafkaEventAdapterServiceDS Successfully deployed the Kafka output event adaptor service
INFO|764/0|"WSO2 Carbon"|17-03-31 10:50:06|[2017-03-31 10:50:06,878] [] ERROR - EventManagementServiceDS Could not create EventProcessorManagementService: Error getting management mode information
INFO|764/0|"WSO2 Carbon"|17-03-31 10:50:06|org.wso2.carbon.event.processor.manager.core.exception.EventManagementException: Error getting management mode information
INFO|wrapper|"WSO2 Carbon"|17-03-31 10:50:06|Trigger found: Exception in line:
INFO|wrapper|"WSO2 Carbon"|17-03-31 10:50:06|org.wso2.carbon.event.processor.manager.core.exception.EventManagementException: Error getting management mode information
INFO|764/0|"WSO2 Carbon"|17-03-31 10:50:06|     at org.wso2.carbon.event.processor.manager.core.internal.CarbonEventManagementService.<init>(CarbonEventManagementService.java:86)
INFO|wrapper|"WSO2 Carbon"|17-03-31 10:50:06|start script scripts/trayMessage.gv
INFO|764/0|"WSO2 Carbon"|17-03-31 10:50:06|     at org.wso2.carbon.event.processor.manager.core.internal.ds.EventManagementServiceDS.activate(EventManagementServiceDS.java:43)
INFO|wrapper|"WSO2 Carbon"|17-03-31 10:50:06|end script scripts/trayMessage.gv
INFO|764/0|"WSO2 Carbon"|17-03-31 10:50:06|     at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
INFO|764/0|"WSO2 Carbon"|17-03-31 10:50:06|     at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
INFO|764/0|"WSO2 Carbon"|17-03-31 10:50:07|     at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
INFO|764/0|"WSO2 Carbon"|17-03-31 10:50:07|     at java.lang.reflect.Method.invoke(Unknown Source)
INFO|764/0|"WSO2 Carbon"|17-03-31 10:50:07|     at org.eclipse.equinox.internal.ds.model.ServiceComponent.activate(ServiceComponent.java:260)
INFO|764/0|"WSO2 Carbon"|17-03-31 10:50:07|     at org.eclipse.equinox.internal.ds.model.ServiceComponentProp.activate(ServiceComponentProp.java:146)
INFO|764/0|"WSO2 Carbon"|17-03-31 10:50:07|     at org.eclipse.equinox.internal.ds.model.ServiceComponentProp.build(ServiceComponentProp.java:345)
INFO|764/0|"WSO2 Carbon"|17-03-31 10:50:07|     at org.eclipse.equinox.internal.ds.InstanceProcess.buildComponent(InstanceProcess.java:620)
INFO|764/0|"WSO2 Carbon"|17-03-31 10:50:07|     at org.eclipse.equinox.internal.ds.InstanceProcess.buildComponents(InstanceProcess.java:197)
INFO|764/0|"WSO2 Carbon"|17-03-31 10:50:07|     at org.eclipse.equinox.internal.ds.Resolver.buildNewlySatisfied(Resolver.java:473)
INFO|764/0|"WSO2 Carbon"|17-03-31 10:50:07|     at org.eclipse.equinox.internal.ds.Resolver.enableComponents(Resolver.java:217)
INFO|764/0|"WSO2 Carbon"|17-03-31 10:50:07|     at org.eclipse.equinox.internal.ds.SCRManager.performWork(SCRManager.java:816)
INFO|764/0|"WSO2 Carbon"|17-03-31 10:50:07|     at org.eclipse.equinox.internal.ds.SCRManager$QueuedJob.dispatch(SCRManager.java:783)
INFO|764/0|"WSO2 Carbon"|17-03-31 10:50:07|     at org.eclipse.equinox.internal.ds.WorkThread.run(WorkThread.java:89)
INFO|764/0|"WSO2 Carbon"|17-03-31 10:50:07|     at org.eclipse.equinox.internal.util.impl.tpt.threadpool.Executor.run(Executor.java:70)
INFO|764/0|"WSO2 Carbon"|17-03-31 10:50:07|Caused by: org.wso2.carbon.event.processor.manager.core.exception.ManagementConfigurationException: event-processor.xmlcannot be found in the path : C:\Users\wso2\Desktop\EI610\ALPHA\wso2ei-6.1.0-ALPHA\wso2\broker\conf\event-processor.xml
INFO|wrapper|"WSO2 Carbon"|17-03-31 10:50:07|Trigger found: Exception in line:
INFO|wrapper|"WSO2 Carbon"|17-03-31 10:50:07|Caused by: org.wso2.carbon.event.processor.manager.core.exception.ManagementConfigurationException: event-processor.xmlcannot be found in the path : C:\Users\wso2\Desktop\EI610\ALPHA\wso2ei-6.1.0-ALPHA\wso2\broker\conf\event-processor.xml
INFO|wrapper|"WSO2 Carbon"|17-03-31 10:50:07|start script scripts/trayMessage.gv
INFO|wrapper|"WSO2 Carbon"|17-03-31 10:50:07|end script scripts/trayMessage.gv
INFO|764/0|"WSO2 Carbon"|17-03-31 10:50:07|     at org.wso2.carbon.event.processor.manager.core.internal.util.ManagementModeConfigurationLoader.loadConfigXML(ManagementModeConfigurationLoader.java:121)
FINEST|764/0|"WSO2 Carbon"|17-03-31 10:50:07|   at org.wso2.carbon.event.processor.manager.core.internal.util.ManagementModeConfigurationLoader.loadManagementModeInfo(ManagementModeConfigurationLoader.java:49)
FINEST|764/0|"WSO2 Carbon"|17-03-31 10:50:07|   at org.wso2.carbon.event.processor.manager.core.internal.CarbonEventManagementService.<init>(CarbonEventManagementService.java:82)
FINEST|764/0|"WSO2 Carbon"|17-03-31 10:50:07|   ... 16 more
FINEST|764/0|"WSO2 Carbon"|17-03-31 10:50:07|Caused by: java.io.FileNotFoundException: C:\Users\wso2\Desktop\EI610\ALPHA\wso2ei-6.1.0-ALPHA\wso2\broker\conf\event-processor.xml (The system cannot find the file specified)
INFO|wrapper|"WSO2 Carbon"|17-03-31 10:50:07|Trigger found: Exception in line:
INFO|wrapper|"WSO2 Carbon"|17-03-31 10:50:07|Caused by: java.io.FileNotFoundException: C:\Users\wso2\Desktop\EI610\ALPHA\wso2ei-6.1.0-ALPHA\wso2\broker\conf\event-processor.xml (The system cannot find the file specified)
INFO|wrapper|"WSO2 Carbon"|17-03-31 10:50:07|start script scripts/trayMessage.gv
FINEST|764/0|"WSO2 Carbon"|17-03-31 10:50:07|   at java.io.FileInputStream.open0(Native Method)
INFO|wrapper|"WSO2 Carbon"|17-03-31 10:50:07|end script scripts/trayMessage.gv
FINEST|764/0|"WSO2 Carbon"|17-03-31 10:50:07|   at java.io.FileInputStream.open(Unknown Source)
FINEST|764/0|"WSO2 Carbon"|17-03-31 10:50:07|   at java.io.FileInputStream.<init>(Unknown Source)
FINEST|764/0|"WSO2 Carbon"|17-03-31 10:50:07|   at org.wso2.carbon.event.processor.manager.core.internal.util.ManagementModeConfigurationLoader.loadConfigXML(ManagementModeConfigurationLoader.java:113)
FINEST|764/0|"WSO2 Carbon"|17-03-31 10:50:07|   ... 18 more
FINEST|764/0|"WSO2 Carbon"|17-03-31 10:50:07|[2017-03-31 10:50:07,722] []  WARN - EventAdapterConfigHelper output-event-adapters.xml can not found in C:\Users\wso2\Desktop\EI610\ALPHA\wso2ei-6.1.0-ALPHA\wso2\broker\conf\output-event-adapters.xml, hence Output Event Adapters will be running with default global configs.
FINEST|764/0|"WSO2 Carbon"|17-03-31 10:50:07|[2017-03-31 10:50:07,722] [] ERROR - EventAdapterConfigHelper Error in converting output-event-adapters.xml to parsed document, hence Output Event Adapters will be running with default global configs.
FINEST|764/0|"WSO2 Carbon"|17-03-31 10:50:10|[2017-03-31 10:50:10,425] []  INFO - EmbeddedRegistryService Configured Registry in 197ms
FINEST|764/0|"WSO2 Carbon"|17-03-31 10:50:10|[2017-03-31 10:50:10,644] []  INFO - RegistryCoreServiceComponent Registry Mode    : READ-WRITE
FINEST|764/0|"WSO2 Carbon"|17-03-31 10:50:14|[2017-03-31 10:50:14,003] []  INFO - SolrClient Default Embedded Solr Server Initialized
FINEST|764/0|"WSO2 Carbon"|17-03-31 10:50:14|[2017-03-31 10:50:14,831] []  INFO - UserStoreMgtDSComponent Carbon UserStoreMgtDSComponent activated successfully.
FINEST|764/0|"WSO2 Carbon"|17-03-31 10:51:47|[2017-03-31 10:51:47,628] []  WARN - CarbonServerManager Carbon initialization is delayed due to the following unsatisfied items:
FINEST|764/0|"WSO2 Carbon"|17-03-31 10:51:47|[2017-03-31 10:51:47,628] []  WARN - CarbonServerManager Waiting for required OSGi Service: org.wso2.carbon.event.publisher.core.EventPublisherService
FINEST|764/0|"WSO2 Carbon"|17-03-31 10:52:47|[2017-03-31 10:52:47,644] []  WARN - CarbonServerManager Carbon initialization is delayed due to the following unsatisfied items:
FINEST|764/0|"WSO2 Carbon"|17-03-31 10:52:47|[2017-03-31 10:52:47,644] []  WARN - CarbonServerManager Waiting for required OSGi Service: org.wso2.carbon.event.publisher.core.EventPublisherService
```

--- a/distribution/src/assembly/bin.xml
+++ b/distribution/src/assembly/bin.xml
@@ -2223,6 +2223,22 @@
 
         <!-- Broker files: Start-->
         <file>
+            <source>
+                ../p2-profile/ei-profile/target/wso2carbon-core-${carbon.kernel.version}/wso2/conf/event-processor.xml
+            </source>
+            <outputDirectory>wso2ei-${pom.version}/wso2/broker/conf</outputDirectory>
+            <fileMode>644</fileMode>
+        </file>
+        <file>
+            <source>
+                ../p2-profile/business-process-profile/target/wso2carbon-core-${carbon.kernel.version}/wso2/conf/output-event-adapters.xml
+            </source>
+            <outputDirectory>${pom.artifactId}-${pom.version}/wso2/broker/conf
+            </outputDirectory>
+            <filtered>true</filtered>
+            <fileMode>644</fileMode>
+        </file>
+        <file>
             <source>src/broker/conf/tomcat/carbon/WEB-INF/web.xml</source>
             <outputDirectory>
                 ${pom.artifactId}-${pom.version}/wso2/broker/conf/tomcat/carbon/WEB-INF/

